internal/provider/feature_resource.go

```diff
@@ -268,15 +262,9 @@ func (r *FeatureResource) Create(ctx context.Context, req resource.CreateRequest
 		return
 	}
 
-	if data.Harness.Skipped.ValueBool() {
-		resp.Diagnostics.AddWarning(fmt.Sprintf("skipping feature [%s] since harness was skipped", data.Id.ValueString()), "given provider runtime labels do not match feature labels")
-		resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
-		return
-	}
-
 	harness, ok := r.store.harnesses.Get(data.Harness.Id.ValueString())
 	if !ok {
-		resp.Diagnostics.AddError("invalid harness id", "how did you get here?")
+		resp.Diagnostics.AddWarning(fmt.Sprintf("skipping feature [%s] since harness was skipped", data.Id.ValueString()), "given provider runtime labels do not match feature labels")
```

Comment:
Is there any risk that we won't have the harness in the map for another reason that's not skipping? Is it worth handling this case here?

Reply:
I have an outstanding PR #137 that hoists the only possible error into the `InventoryData`. This PR doesn't depend on #137 but in the near future I hope to make it so most of the `inventory/store` logic is abstracted away.

so tl;dr is no, the only way we would a) be in a `feature_resource` and b) not have a valid harness is if the harness is skipped, everywhere else would fail before reaching the `feature` 🤞